sim/xcelium/defaults.yml

```diff
@@ -0,0 +1,25 @@
+sim.xcelium:
+  version: "XCELIUM2103"
+
+  xcelium_bin: "${cadence.cadence_home}/XCELIUM/${sim.xcelium.version}/tools/xcelium/bin/64bit/xrun"  
+  xcelium_bin_meta: lazysubst
+
+  xmsimrc_def: "${cadence.cadence_home}/XCELIUM/${sim.xcelium.version}/tools/xcelium/files/xmsimrc"
+  xmsimrc_def_meta: lazysubst
+
+
+  xcelium_home: "${cadence.cadence_home}/XCELIUM/${sim.xcelium.version}"
+  xcelium_home_meta: lazysubst
+
+  # XRUN Top-Level Customization 
+  enhanced_recompile: True
+  xmlibdirname: null
+  xmlibdirpath: null
+  simtmp: null
+  snapshot: null
+  global_access: False
+  compile_opts: null
```

Comment:
Are you able to use any of the `sim.inputs.complier_cc_opts` and `sim.inputs.compiler_ld_opts`?

Reply:
This took a while to figure out, but can be supported by using -Wcc and -Wld opts in xrun.  There is a -Wcxx for Cpp compiler options. Should we consider adding in a separate key for this?

Reply:
Should add that compile_opts is meant for general xrun compilation options, not c compilation options specifically.

Reply:
Actually, the way that Chipyard is structured just passes Cpp compiler options to VCS as `-CFLAGS`: https://github.com/ucb-bar/chipyard/blob/32b5e8ad844f3138a8901be40e44ad226752d69d/sims/common-sim-flags.mk#L6 and https://github.com/ucb-bar/chipyard/blob/32b5e8ad844f3138a8901be40e44ad226752d69d/vcs.mk#L33

Is what you can pass in as `-Wcxx` as superset of `-Wcc`?

Reply:
Don't think `Wcxx` is a superset, but looking at the Chipyard structure I think we should be using `Wcxx` instead.